ext/ruby.c

```diff
@@ -5,6 +5,42 @@ VALUE RS_Qtrue = Qtrue;
 VALUE RS_Qfalse = Qfalse;
 VALUE RS_Qnil = Qnil;
 
+// begin ruby_value_type
+int RS_T_NONE = RUBY_T_NONE;
+
+int RS_T_OBJECT = RUBY_T_OBJECT;
+int RS_T_CLASS = RUBY_T_CLASS;
+int RS_T_MODULE = RUBY_T_MODULE;
+int RS_T_FLOAT = RUBY_T_FLOAT;
+int RS_T_STRING = RUBY_T_STRING;
+int RS_T_REGEXP = RUBY_T_REGEXP;
+int RS_T_ARRAY = RUBY_T_ARRAY;
+int RS_T_HASH = RUBY_T_HASH;
+int RS_T_STRUCT = RUBY_T_STRUCT;
+int RS_T_BIGNUM = RUBY_T_BIGNUM;
+int RS_T_FILE = RUBY_T_FILE;
+int RS_T_DATA = RUBY_T_DATA;
+int RS_T_MATCH = RUBY_T_MATCH;
+int RS_T_COMPLEX = RUBY_T_COMPLEX;
+int RS_T_RATIONAL = RUBY_T_RATIONAL;
+
+int RS_T_NIL = RUBY_T_NIL;
+int RS_T_TRUE = RUBY_T_TRUE;
+int RS_T_FALSE = RUBY_T_FALSE;
+int RS_T_SYMBOL = RUBY_T_SYMBOL;
+int RS_T_FIXNUM = RUBY_T_FIXNUM;
+int RS_T_UNDEF = RUBY_T_UNDEF;
+
+int RS_T_IMEMO = RUBY_T_IMEMO;
+int RS_T_NODE = RUBY_T_NODE;
+int RS_T_ICLASS = RUBY_T_ICLASS;
+int RS_T_ZOMBIE = RUBY_T_ZOMBIE;
+
+int RS_T_MASK = RUBY_T_MASK;
```

Comment:
discussed with @wycats, it's best to remove this one for now. if we do bind it in the future (unlikely that it's necessary), we would probably want to bind it as an integer rather than as a `ruby_value_type`.

Reply:
:+1: